### ISSUE TITLE
setOption

### ISSUE THREAD

**Issue by marty-buly** [stefanuebe/vaadin_fullcalendar#3]
Hi Stefan,

good job 👍. Thank you for providing this plugin.

I have a question regarding 'options' of FullCalendar.io and way how you define it in your FullCalendar.class. Currently there is limited set of options you can pass down to the FullCalendar instance and this is set by you inner class Option. Do you plan to provide 'all' possible options for FullCalendar in this way? Looks to me a bit limiting (but I might be missing a correct information why you made it this way. 
Basically when I want to set something additional, I have subclassed your FullCalendar.class and provided non-limiting function like:
```
	public void setOption(Serializable option, Serializable value) {
		getElement().callFunction("setOption", option, value);
	}
```
This is OK for me, but I can imagine that there is no need for me to do any subclassing if you could factor setOption method in some similar way as above.

What do you think?
Thnx for your thoughts.
Martin

**Comment by stefanuebe** [comment]
Hi, 

you're right, did not had have that in mind to provide a more generic method. Will take that over into the next release, thank you :)

**Comment by stefanuebe** [comment]
Part of 1.2.0.
